### ISSUE TITLE
Please add Gitlens Fork before it was owned by GitKraken

### ISSUE THREAD

**Issue by lamyergeier** [open-vsx/publish-extensions#680]
This extension is available on vscode at : [[GitLess - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=maattdd.gitless)](https://marketplace.visualstudio.com/items?itemName=maattdd.gitless)

[[maattdd/vscode-gitlens: Supercharge the Git capabilities built into Visual Studio Code — Visualize code authorship at a glance via Git blame annotations and code lens, seamlessly navigate and explore Git repositories, gain valuable insights via powerful comparison commands, and so much more](https://github.com/maattdd/vscode-gitlens)](https://github.com/maattdd/vscode-gitlens)

**Comment by filiptronicek** [comment]
Hi there @lamyergeier 👋!

I think there is no harm in one-time publishing this extension, but does it really do what you suggest it does? I see GitKraken links in this version already 🤔 
<img width="2032" alt="image" src="https://user-images.githubusercontent.com/29888641/236807143-22ee7541-359e-45db-a6f1-be9d92161253.png">

**Comment by lamyergeier** [comment]
Yes there are Gitkraken links but all features are free upto that git commit.

**Comment by lamyergeier** [comment]
@filiptronicek  It is not correct. When installed in Codium  it shows extension version as v12. While when I install the extension in Microsoft VSCode it shows v11.7 as follows

![image](https://github.com/open-vsx/publish-extensions/assets/42092626/cce01631-8ec8-4ed3-90c5-6d9d8df43474)

**Comment by filiptronicek** [comment]
@lamyergeier ok, I can unpublish this version. Do you know where the extension would be published from exactly? I don't see any difference in the fork - no commits added by @maattdd. I'm looking for the actual source code from which they published it.

**Comment by GitMensch** [comment]
Hm, I also see that - the version in the marketplace just doesn't have any matching branch / tag in the repo - so there is no use in the extra publishing from the repo.

Also open-vsx already include the "completely free" version https://open-vsx.org/extension/eamodio/gitlens/11.7.0 (and with the missing additional commits and an "active" fork where changes are actually done (for example each change from the official version that doesn't change anything in the "plus" directory could be merged in - because this is the _sole_ directory that has a different license.
